File: Questions/Day-1/Java/Q3.java
import java.util.Scanner;

public class Q3 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        System.out.print("Enter a: ");
        int a = in.nextInt();

        System.out.print("Enter b: ");
        int b = in.nextInt();

        System.out.println("\nAddition of a and b is "+ (a+b));
        System.out.println("\nSubtraction of a and b is "+ (a-b));
        System.out.println("\nDivision of a and b is "+ (a/b));
        System.out.println("\nMultiplication of a and b is "+ (a*b));
    }
}

File: Questions/Day 1/cpp/Q3.cpp
#include <iostream>
using namespace std;

int main()
{
    int a = 0;
    cout << "Enter a: ";
    cin >> a;

    int b = 0;
    cout << "Enter b: ";
    cin >> b;

    cout << "\nAddition of a and b is " << (a + b);
    cout << "\nSubtraction of a and b is " << (a - b);
    cout << "\nDivision of a and b is " << (a / b);
    cout << "\nMultiplication of a and b is " << (a * b);

    return 0;
}